ConsoleAppHomeworkMethods/ConsoleAppHomeworkMethods/Program.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Security.Cryptography;
+
+/*int MaxValue (int x, int y)
+{
+    return Math.Max (x, y);
+}
+int x, y;
+Console.Write("Add value x: ");
+bool isValueX = int.TryParse(Console.ReadLine(), out x);
+
+Console.Write("Add value y: ");
+bool isValueY = int.TryParse(Console.ReadLine(),out y);
+ int result = MaxValue (x, y);
+
+Console.WriteLine( "Max value is : " + result);
+
+int MinValue (int x, int y)
+{
+    return Math.Min (x, y);
+}
+
+Console.Write("Add value x: ");
+bool isValueX1 = int.TryParse(Console.ReadLine(), out x);
+
+Console.Write("Add value y: ");
+bool isValueY1 = int.TryParse(Console.ReadLine(), out y);
+int result1 = MinValue(x, y);
+
+Console.WriteLine("Min value is : " + result1);*/
+
+/*
+int x, y;
+
+Console.Write("Add value x: ");
+bool isValueX2 = int.TryParse(Console.ReadLine(), out x);
+
+Console.Write("Add value y: ");
+bool isValueY2 = int.TryParse(Console.ReadLine(), out y);
+bool TrySumm(int x , int y)
+{
+    if (x % 2 != 0 && y % 2 != 0)
+    {
+        return true; 
+    }
+    else
+    { 
+        return false;
+    }
+        
+}
+bool isSumm = TrySumm (x,y);
+
+if (isSumm)
+{
+    Console.WriteLine(" Sum  = " + (x + y ));
+}
+else
+{
+    Console.WriteLine(" Incorrect values");
+}*/
+/*int x, y;
+Console.Write("Add value x: ");
+bool isValueX2 = int.TryParse(Console.ReadLine(), out x);
+
+Console.Write("Add value y: ");
+bool isValueY2 = int.TryParse(Console.ReadLine(), out y);
+bool Sumodd(int x, int y, out int summ)
+{
+    summ = x + y;
+    if ((x + y) % 2 != 0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+    
+}
+bool isSumm = Sumodd(x,y, out int summ);
+Console.WriteLine(summ);*/
+
+/*
+int x, y;
+Console.Write("Add value x: ");
+bool isValueX2 = int.TryParse(Console.ReadLine(), out x);
+
+Console.Write("Add value y: ");
+bool isValueY2 = int.TryParse(Console.ReadLine(), out y);
+bool Sumodd(int x, int y, out int Max)
+{
+    Max = Math.Max(x,y);
+    if (x % 2 != 0 && y % 2 != 0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+
+}
+bool isSumm = Sumodd(x, y, out int Max);
+Console.WriteLine(Max); */
+/*
+int x, y;
+Console.Write("Add value x: ");
+bool isValueX2 = int.TryParse(Console.ReadLine(), out x);
+
+Console.Write("Add value y: ");
+bool isValueY2 = int.TryParse(Console.ReadLine(), out y);
+bool Sumodd(int x, int y, out int Min)
+{
+    Min = Math.Min(x, y);
+    if (x % 2 != 0 && y % 2 != 0)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+
+}
+bool isSumm = Sumodd(x, y, out int Min);
+Console.WriteLine(Min);*/
+
+/*
+ string Repeat (string  text , int times)
+{
+    string newText = " ";
```

Comment:
When we concatenate strings in cycle multiple times, its better to use StringBuilder class. But the logic looks good :)